power-control-x86/src/power_control.cpp

```diff
@@ -1927,6 +1917,10 @@ int main(int argc, char* argv[])
     {
         return -1;
     }
+    else
```

Comment:
YAAP should start once BMC ready is signaled. Hence no need for "else" condition